subprojects/core/src/main/java/org/gradle/caching/configuration/AbstractBuildCache.java

```diff
@@ -59,4 +62,22 @@ public boolean isPush() {
     public void setPush(boolean push) {
         this.push = push;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Nullable
```

Comment:
This has changed quite a lot since the last commit, but essentially Luke decided to keep that nullability in the correspondent place: `If the factory doesn’t supply a type value, the build operation result field for this should be null. i.e. we’ll do the defaulting on the scans server.`